buffer/src/main/java/io/netty/buffer/api/BufferStub.java

```diff
@@ -458,4 +459,19 @@ public Buffer touch(Object hint) {
         delegate.touch(hint);
         return this;
     }
+
+    @Override
+    public String toString(Charset charset) {
```

Comment:
Can we add a JavaDoc here to help people know that this method converts `'Buffer's` data into `String`?

Reply:
This is not needed as its already done in the interface. (see above).